common4j/src/main/com/microsoft/identity/common/java/cache/SharedPreferencesAccountCredentialCache.java

```diff
@@ -224,14 +268,17 @@ public boolean test(String value) {
                 );
 
                 if (null == account) {
-                    Logger.warn(TAG, ACCOUNT_RECORD_DESERIALIZATION_FAILED);
+                    Logger.warn(methodTag, ACCOUNT_RECORD_DESERIALIZATION_FAILED);
+                } else if (EMPTY_ACCOUNT.equals(account)) {
+                    Logger.warn(methodTag, "The returned Account was uninitialized. Removing...");
```

Comment:
I'm wondering why might this ever happen. Have you seen this happen in your testing?